fe/fe-core/src/main/java/com/starrocks/server/OlapTableFactory.java

```diff
@@ -215,7 +215,6 @@ public Table createTable(LocalMetastore metastore, Database db, CreateTableStmt
                 }
                 String storageVolumeId = svm.getStorageVolumeIdOfTable(tableId);
                 metastore.setLakeStorageInfo(table, storageVolumeId, properties);
-                useFastSchemaEvolution = false;
```

Comment:
should keep this `false` until the feature is done.

Reply:
Updated